Gemfile

```diff
@@ -166,7 +166,7 @@ group :development do
 
   # Simple command execution over SSH. Lightweight deployment tool.
   # Using our own version until https://github.com/mina-deploy/mina/pull/361 is merged.
-  gem 'mina', git: "git@github.com:samfundet/mina.git"
+  gem 'mina', git: "git://github.com/Samfundet/mina.git"
```

Comment:
When using the git@github syntax one requires your ssh key to be added to the relevant github account, even though the repository is open source.
The git protocol simply clones down the git file at that location.